Email-classification.ipynb

```diff
@@ -0,0 +1,874 @@
+{
```

Comment:
The notepad looks quite detailed - good job. I haven't ran it locally I will do it on next week but the information provided is quite good. The only thing I've found is the strange value of test data like ```with test size = 0.15%:```. Maybe you mean 15% there?